EnRonSwanson/BangazonCLI
# Mb editprod interface

## Fixes #8 .

## Changes proposed in this pull request: 
- Program interface with methods to get products for customer and update product attributes indicated in ticket: Title, Description, Price, QuantityAvailable
- seed data dates are same format as default DateTime.Now format 

## How to test: 
- delete test db if you have one 
- dotnet run
- following onscreen commands- add a customer, choose it as active, add a product, edit the product
- ensure database values have changed to what you've edited them for
- pay special attention to the datetime formats, as they were the most finicky

@EnRonSwanson/
